test/linter-test.js

```diff
@@ -207,6 +242,21 @@ describe('linter', () => {
           assert.equals(out, 'console.log(\'!\');\n');
         });
 
+        describe('--fix-dry-run', () => {
+
+          if (checkVersion(eslint_version, '4.9')) {
```

Comment:
I'd prefer conditional tests to be skipped like so:

```js
before(function () {
  if (checkVersion(eslint_version, '4.9')) {
    this.skip();
  }
});
```

Reply:
Unfortunately, I can not do it everywhere. If I do it in the tests, then it fails. I can add other `describe` and then use `before` but it looks too complicated. If you wish, I can do it.

Reply:
Yeah, I see what you mean. Let's leave it as is. Thank you!